src/utils/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+add_subdirectory(boot_struct)
+#add_subdirectory(build_info)
+add_subdirectory(cqueue)
+add_subdirectory(crash_catcher)
+add_subdirectory(crc16)
+add_subdirectory(crc32)
+add_subdirectory(encrypt)
+add_subdirectory(heap)
+add_subdirectory(hexdump)
+add_subdirectory(hwtimer_list)
+add_subdirectory(intersyshci)
+add_subdirectory(kfifo)
+#add_subdirectory(libc)
+add_subdirectory(list)
+add_subdirectory(lockcqueue)
+add_subdirectory(retention_ram)
+#add_subdirectory(rom_utils)
+#add_subdirectory(sha256)
```

Comment:
Should this be commented out?

Reply:
Line 18.

Reply:
Probably not, seems to be an oversight, though with that said, the sha256 target should also have
`target_link_libraries(util_sha256 platform_cmsis platform_hal)`

Reply:
Could you fix that for us? Thanks.